<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanWidget</class>
 <widget class="QWidget" name="PlanWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>181</width>
    <height>372</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>181</width>
    <height>372</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>181</width>
    <height>372</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#wContainer
{
  border-radius: 3px;
  background-color: #ffffff;
  border: 1px solid;
  border-color: rgba(0, 0, 0, 0.1);
}
#wContainer:hover
{
  border-radius: 3px;
  border: 1px solid;
  border-color: rgba(0, 0, 0);
}

#wPrice
{
  background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgba(249,249,249,0.3), stop: 1 rgba(238,238,238,0.3));
 border-top: 1px solid;
 border-top-color: rgba(0, 0, 0, 0.1);
 border-bottom: 1px solid;
 border-bottom-color: rgba(0, 0, 0, 0.1);
}

QPushButton
{
  border: none;
}

#lTip
{
  font-family: SourceSansPro-Regular;
  font-size: 14px;
  color: #fe2229;	
}

#lProPlan
{
  font-family: SourceSansPro-Regular;
  font-size: 20px;
  color: #666666;
}

#lPeriod
{
  font-family: SourceSansPro-Regular;
  font-size: 20px;
  color: #ff333a;
}

#lPrice
{
  font-family: SourceSansPro-Regular;
  color: #333333;
}

#lStorageInfo, #lBandWidthInfo
{
  font-family: SourceSansPro-Regular;
  font-size: 16px;
  color: #333333;
}
#lStorage, #lBandWidth
{
  font-family: SourceSansPro-Regular;
  font-size: 13px;
  letter-spacing: 0px;
  color: #999999; 
}

</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wContainer" native="true">
     <property name="minimumSize">
      <size>
       <width>181</width>
       <height>372</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>181</width>
       <height>372</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="wCrest" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lTip">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string>popular!</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>13</width>
               <height>17</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="bcrest">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="Resources_linux.qrc">
               <normaloff>:/images/proicrest.png</normaloff>:/images/proicrest.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>58</width>
               <height>58</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>13</width>
               <height>17</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lProPlan">
        <property name="text">
         <string>PRO LITE</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wPrice" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>97</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>97</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lPrice">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>4.99 EUR</string>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lPeriod">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>/month</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wPlanInfo" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>136</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>136</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>25</number>
           </property>
           <property name="leftMargin">
            <number>15</number>
           </property>
           <item>
            <widget class="QPushButton" name="bStorage">
             <property name="minimumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="Resources_linux.qrc">
               <normaloff>:/images/storage.png</normaloff>:/images/storage.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>32</height>
              </size>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="lStorageInfo">
                <property name="text">
                 <string>200 GB</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lStorage">
                <property name="text">
                 <string>STORAGE</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>25</number>
           </property>
           <property name="leftMargin">
            <number>15</number>
           </property>
           <item>
            <widget class="QPushButton" name="bBandwidth">
             <property name="minimumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="Resources_linux.qrc">
               <normaloff>:/images/bwupgrade.png</normaloff>:/images/bwupgrade.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_4" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>32</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>32</height>
              </size>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="lBandWidthInfo">
                <property name="text">
                 <string>1 TB</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lBandWidth">
                <property name="text">
                 <string>BANDWIDTH</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources_linux.qrc"/>
 </resources>
 <connections/>
</ui>
